### DISCUSSION TITLE
Feature request: Allow file access via URL

### DISCUSSION THREAD

**Discussion by wq9578** [ZUGFeRD/mustangproject#359] [Ideas]
It would be desirable to also allow specifying an URL instead of a local filename (standard behaviour in PHP):
```
java -jar Mustang-CLI-2.10.0.jar --no-notices --disable-file-logging --action validate --source https://www.mustangproject.org/files/MustangGnuaccountingBeispielRE-20201121_508.pdf
```

By the way:
- Update for https://www.mustangproject.org/einvoices/#calculation: current version of 16931-1 available at https://www.beuth.de/de/norm/din-en-16931-1/327729047
- Update for https://www.mustangproject.org/xrechnung/#xrechnung: current version 3.0.1 since Feb 1st: https://xeinkauf.de/app/uploads/2023/09/301-XRechnung-2023-09-22.pdf

**Comment by jstaerk** [comment]
Thanks I updated the page accordingly. Regarding your request:

- I can show you how to implement it if you like, 
- I can do it for money (should not be much we're talking three digit here) 

but in any case I suggest to read the invoice from stdin, which would then allow the command to be piped with e.g. curl or wget and would be less complex and more versatile.
